style.css

```diff
@@ -552,6 +603,261 @@ i {
   color: #fff;
 }
 
+.card {
+  width: 100%;
+}
+
+.card-title {
+  color: #172b4d;
+  font-weight: 700;
+}
+
+#popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #c1c7d0;
+  z-index: 10;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  opacity: 0.97;
+  overflow: scroll;
+}
```

Comment:
- [ ] Good job so far :clap: However,  On your  `popup` card id for both mobile and desktop,  Please ensure that the font also matches the given design requirement that says `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using
[this design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-418&mode=design&t=Doc6SdyUeETs3nNw-0)
- [ ] Please ensure that you remove the `Lorem Epsum` texts and replace them with your customize texts.


below is an illustration of the difference between yours  and the given design :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Desktop](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/43528594/fc69b1ed-5469-45e0-b43a-89148b18d8f7)


</td>
      <td>

![Figma Desktop](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/43528594/53c492f5-b22c-4f5f-b4fd-470aea1f8c9f)


</td>
    </tr>
  </table> 


<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Mobile](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/43528594/b3a51a14-dc08-4ca5-be3d-8d417fbf3fa0)


</td>
      <td>

![Figma Mobile Design](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/43528594/c694524f-a7ae-44bd-9c40-c2d101f7d51c)


</td>
    </tr>
  </table> 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Canopy mobile](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/43528594/396cab3c-df26-4644-8900-01572c7569c9)


</td>
      <td>

![Figma mobile Canopy](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/43528594/1e197054-d2d6-464a-a523-2a1391fe41f0)


</td>
    </tr>
  </table>